examples/projections/nongeo/cartesian_linear.py

```diff
@@ -2,8 +2,11 @@
 Cartesian linear
 ================
 
-**X**\ *width*/[*height*]: Give the *width* of the figure and the optional
-*height*.
+**X**\ *width*\ [/*height*] or **x**\ *x-scale*\ [/*y-scale*]
+
+Give the *width* of the figure and the optional *height*.
+The lower-case version **x** is similar to **X** but expects
+a *x-scale* and an optional *y-scale*.
```

Comment:
```suggestion
an *x-scale* and an optional *y-scale*.
```

Reply:
These typo fixes are included in https://github.com/GenericMappingTools/pygmt/pull/2643/commits/943ad7fe6f80db3e2bdb320d0f1d46c12802a29c, in which I added some text to mention the possibility of using geographic as well as data/time input data.